lte/gateway/c/core/oai/common/log.c

```diff
@@ -1509,12 +1509,10 @@ const char* get_short_file_name(const char* const source_file_nameP) {
 }
 
 // Return the hex representation of a char array
-
 char* bytes_to_hex(char* byte_array, int length, char* hex_array) {
   int i;
   for (i = 0; i < length; i++) {
     sprintf(hex_array + i * 3, " %02x", (unsigned char)byte_array[i]);
   }
```

Comment:
@ardzoht, can we replace this `sprintf` with `snprintf`?

Reply:
will update the PR, I also think this function should live under `conversions.h/c`, so will move it there as well.